TestAssets/TestProjects/VSTestDotNetCore/VSTestDotNetCore.csproj

```diff
@@ -3,7 +3,7 @@
   <PropertyGroup>
```

Comment:
Can these be updated to point to `testAsset.props`?  That way they are consistent with every other test asset file.